common/djangoapps/track/middleware.py

```diff
@@ -18,7 +18,11 @@
     'REMOTE_ADDR': 'ip',
     'SERVER_NAME': 'host',
     'HTTP_USER_AGENT': 'agent',
-    'PATH_INFO': 'path'
+    'PATH_INFO': 'path',
+    # Not a typo. See:
+    # http://en.wikipedia.org/wiki/HTTP_referer#Origin_of_the_term_referer
+    'HTTP_REFERER': 'referer',
+    'HTTP_ACCEPT_LANGUAGE': 'accept_language'
```

Comment:
This is a nitpick, but it'd be good practice to end this line with a trailing comma. That way, when new keys are added to this dictionary in the future, this line doesn't need to show up in the diff.